oak-run-commons/src/test/java/org/apache/jackrabbit/oak/index/AbstractIndexTestCommand.java

```diff
@@ -16,30 +16,28 @@
  * specific language governing permissions and limitations
  * under the License.
  */
-
 package org.apache.jackrabbit.oak.index;
 
-import java.io.File;
-import java.io.IOException;
+import org.apache.jackrabbit.oak.plugins.index.IndexConstants;
+import org.junit.After;
+import org.junit.Rule;
+import org.junit.rules.TemporaryFolder;
 
 import javax.jcr.Node;
 import javax.jcr.PropertyType;
 import javax.jcr.RepositoryException;
 import javax.jcr.Session;
-
-import org.apache.jackrabbit.oak.plugins.index.IndexConstants;
-import org.apache.jackrabbit.oak.plugins.index.lucene.util.LuceneIndexDefinitionBuilder;
-import org.junit.After;
-import org.junit.Rule;
-import org.junit.rules.TemporaryFolder;
+import java.io.File;
+import java.io.IOException;
```

Comment:
Same as above comments.

Reply:
@rishabhdaim  -if you are referring to order of imports - that gets changes usually by IDE formatting (usually for the better). I don't think it's doing any harm.